addon-test-support/-private/utils.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export function checkMatcher(message, matcher) {
+  if (typeof matcher === 'string') {
+    return message.includes(matcher);
```

Comment:
My gut feeling would be that this should be `message === matcher` (e.g. the default is that you are matching the exact string, not a subset of it).

What do you think?